front/app/components/SeatInfo/TrackerInfo.test.tsx

```diff
@@ -103,7 +103,7 @@ describe('TrackerInfo', () => {
   it('shows correct collaborators additional seats when user has used more', () => {
```

Comment:
```suggestion
  it('shows correct moderator additional seats when user has used more', () => {
```

Let's do something similar on line 92